virttest/qemu_devices/qdevices.py

```diff
@@ -1105,6 +1116,35 @@ def verify_hotplug(self, out, monitor):
         else:
             return False
 
+    def _cmdline_json(self):
+        command_dict = {}
+        out = "-%s " % self.type
+        for key, val in six.iteritems(self.params):
+            # wwn needs to be presented as hexadecimal
+            if key in ["wwn"]:
+                command_dict[key] = int(val, 16)
+            # physical_block_size from device ("driver": "scsi-hd")
+            # logical_block_size from device ("driver": "scsi-hd")
+            # bootindex from device ("driver": "scsi-hd")
+            # max_sectors from device ("driver": "virtio-scsi-pci")
+            # num_queues from device ("driver": "virtio-scsi-pci")
+            # virtqueue_size from device ("driver": "virtio-scsi-pci")
+            elif key in ["physical_block_size", "logical_block_size",
+                         "bootindex", "max_sectors", "num_queues",
+                         "virtqueue_size"]:
+                command_dict[key] = int(val.split(" ")[0])
```

Comment:
why we need a split(" ") here? the space chars should be removed and then assigned to cartesian param

Reply:
> why we need a split(" ") here? the space chars should be removed and then assigned to cartesian param

While I debugging the code, I had found some space chars left at param. That's why those code is here.
However, I've tried the code `command_dict[key] = int(val)` works fine as well.